assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -58,7 +58,7 @@ const Label = styled.span`
   @media ${({ theme }) => theme.breakpoint.desktop} {
     display: block;
     min-width: 115px;
-    text-align: left;
+    text-align: ${({ isRTL }) => (isRTL ? 'right' : 'left')};
```

Comment:
I don't think this is needed.

Our stylis RTL plugin _should_ automatically replace `text-algin: left` with `text-align: right` (like RTLCSS does).

But don't take my word for it :-)

Reply:
ope yeaup. added all the consideration for rtl manually, then realized in the canvas we're opting out of the stylis plugin. so I moved it out of the canvas & removed all the manual rtl stuff. Must of missed this when I was cleaning it up. thanks for catching!